client/dashboard/leaderboards/top-customers.js

```diff
@@ -0,0 +1,125 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { map } from 'lodash';
+
+/**
+ * WooCommerce dependencies
+ */
+import { formatCurrency, getCurrencyFormatDecimal } from '@woocommerce/currency';
+import { getNewPath, getPersistedQuery } from '@woocommerce/navigation';
+import { Link } from '@woocommerce/components';
+import { numberFormat } from '@woocommerce/number';
+
+/**
+ * Internal dependencies
+ */
+import Leaderboard from 'analytics/components/leaderboard';
+
+export class TopCustomers extends Component {
+    constructor( props ) {
+        super( props );
+
+        this.getRowsContent = this.getRowsContent.bind( this );
+        this.getHeadersContent = this.getHeadersContent.bind( this );
+    }
+
+    getHeadersContent() {
+        return [
+            {
+                label: __( 'Customer Name', 'woocommerce-admin' ),
+                key: 'name',
+                required: true,
+                isLeftAligned: true,
+                isSortable: true,
```

Comment:
I think we'll actually want to turn all these `isSortable` values to `false` since the leaderboards should remain in order.